staging/src/k8s.io/apimachinery/pkg/util/net/util_test.go

```diff
@@ -66,3 +69,28 @@ func TestIPNetEqual(t *testing.T) {
 		}
 	}
 }
+
+func TestIsConnectionRefused(t *testing.T) {
```

Comment:
is it possible to actually exercise getting a connection refused error from the network stack? (I'm not sure it is possible to do reliably in a unit test)

Reply:
I think it is hard to actually exercise getting a connection refused error from the network stack in a unit test. I found this bug just because when i restart the kube-apiserver, an connection refused error accurred but can't be recognized.